packages/salesforcedx-utils-vscode/src/services/sourceTrackingService.ts

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2023, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { Connection, Org, SfProject } from '@salesforce/core';
+import {
+  SourceTracking,
+  SourceTrackingOptions
+} from '@salesforce/source-tracking';
+
+export class SourceTrackingService {
+  public static async createSourceTracking(
+    projectPath: string,
+    connection: Connection
+  ): Promise<SourceTracking> {
+    const project = await SfProject.resolve(projectPath);
+    const org = await Org.create({ connection });
+    const options: SourceTrackingOptions = {
+      org,
+      project,
+      ignoreLocalCache: false,
```

Comment:
I set this to false per the docs, but I occasionally saw some issues with cacheing during dev.  QA should include testing with creating new files etc to see if any cacheing issues can be found.

<img width="500" alt="2023-03-01_07-10-19" src="https://user-images.githubusercontent.com/46458081/222180307-61afc0a9-b2c8-4f65-b0b4-4a2fdd48dbc2.png">